app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
To clarify, `try(:downcase)` is the scary bit, do you often have servers without a vendor field?

For comparison, this is what the other decorators do:

```
app/decorators/host_decorator.rb
7:    "svg/vendor-#{vmm_vendor_display.downcase}.svg"

app/decorators/manageiq/providers/configuration_manager_decorator.rb
8:      "svg/vendor-#{image_name.downcase}.svg"

app/decorators/manageiq/providers/automation_manager_decorator.rb
8:      "svg/vendor-#{image_name.downcase}.svg"

app/decorators/vm_or_template_decorator.rb
7:    "svg/vendor-#{vendor.downcase}.svg"

app/decorators/ext_management_system_decorator.rb
7:    "svg/vendor-#{image_name}.svg"
```